pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -86,6 +87,7 @@ const (
 // rejectRequest sends reject response to the requesting client, based on the
 // packet-in message.
 func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
+	// All src/dst means the source/destination of reject packets.
```

Comment:
means -> mean

How about adding "which are destination/source of the incoming packet"? Then change "reject packets" to "the reject packet".

Reply:
Make sense. Fixed.